gradle.properties

```diff
@@ -15,12 +15,13 @@ releaseRepositoryUrl=https://oss.sonatype.org/service/local/staging/deploy/maven
 snapshotsRepositoryUrl=https://oss.sonatype.org/content/repositories/snapshots/
 ###############################################################################################
 # Localized gradle settings to ensure modules can be loaded/configured as quickly as possible.
-################################################################################################
+###############################################################################################
 org.gradle.configureondemand=true
 org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.jvmargs=-Xms1024m -Xmx4048m -XX:SoftRefLRUPolicyMSPerMB=0 -noverify -XX:TieredStopAtLevel=1
 org.gradle.unsafe.configuration-cache=false
+dependencyCacheDays=0
```

Comment:
Problem with this approach is that the environment CI runs to verify tests is different from that of a regular developer. I think the goal should be to try to make sure there are as few diffs as possible, to make troubleshooting easier. If I am seeing a different behavior between my env and CI, it will be incredibly difficult to see that my tests pass because I am on a latest version of something and CI is running a cache dependency.

In general, I am against running "cached" dependencies in a CI environment as a basis for tests. It makes tests unpredictable, and moot. If there are issues with an environment that require this, we should probably review and fix those first before using this as a band-aid.

Reply:
The other issue is that while we only have to put this change to make some MacOS thing work, it in fact affects every other environment and developer. So the fix is intrusive. 

I would prefer to figure out what exactly the issue with MacOS builds are, and remove/fix that, or use the same exact setting for all environments, and a value that is reasonable small, like 1, which seems like it would be a good compromise.

Reply:
The default is 1 day for gradle, and this only impacts snapshots because I don't think we have any dynamic versions. The one snapshot that CAS has currently rarely ever changes but I think using it for a day is fine. If we don't allow the snapshot to be cached for a day then it will be downloaded 250+ times assuming it is a dependency for all the tests, and it only takes a small percentage of those to fail for it to break a lot of tests.

Reply:
Yup I think one or two is fine.